### PR TITLE
#102 - fix bug on empty DjangoQ

### PR DESCRIPTION
**Pull request by shinneider**
Closes issue [102](https://github.com/stphivos/django-mock-queries/issues/102)

### CODE REVIEW

File: tests/test_query.py [hunk 1]
```diff
@@ -1240,3 +1240,12 @@ def test_query_datetimes_second(self):
...
         assert not bool(qs)
+
+    def test_empty_query(self):
+        car1 = Car(id=101)
+        car2 = Car(id=102)
+
+        mockset = MockSet(car1, car2)
+        self.assertEqual(mockset.count(), 2)
+        self.assertEqual(mockset.filter(Q()).count(), 2)
+        self.assertEqual(mockset.filter(Q(id=2)).count(), 1)
```
Comment (stphivos) [inline tests/test_query.py:1251]: `AssertionError: 0 != 1`

What is the intention of the last assert?

Btw you can use the same test command as travis from [tox.ini](https://github.com/stphivos/django-mock-queries/blob/master/tox.ini#L31) to verify it works:
```
py.test django_mock_queries/ tests/ --cov-report term-missing --cov=django_mock_queries --flake8
```

**Review by stphivos — CHANGES_REQUESTED** [review]
Thanks! Just left a comment for the failing test: https://travis-ci.org/github/stphivos/django-mock-queries/jobs/754813071

File: tests/test_query.py (hunk 1 above)
Comment (shinneider) [reply tests/test_query.py]: I removed this,
isn't necessary

File: tests/test_query.py [hunk 2]
```diff
@@ -1240,3 +1240,12 @@ def test_query_datetimes_second(self):
...
         assert not bool(qs)
+
+    def test_empty_queryset_filter(self):
+        car1 = Car(id=101)
+        car2 = Car(id=102)
+
+        mockset = MockSet(car1, car2)
+        self.assertEqual(mockset.count(), 2)
+        self.assertEqual(mockset.filter(Q()).count(), 2)
+
```
Comment (stphivos) [inline tests/test_query.py:1251]: flake8 lint error "blank line at end of file" :)

https://travis-ci.org/github/stphivos/django-mock-queries/jobs/754837922

**Review by stphivos — CHANGES_REQUESTED** [review]

**Pull request closed without merging**